package-lock.json

```
@@ -52,7 +52,13 @@
       "engines": {
         "node": "^16.13.0",
         "npm": "^8.1.0",
-        "npx": "^8.1.0"
+        "npx": "^8.1.0",
+        "pnpm": "^7.14.1",
+        "yarn": "^1.22.19"
+      },
+      "peerDependencies": {
+        "@types/markdown-it": "*",
+        "markdown-it": "*"
```

```suggestion
      "peerDependencies": {
        "@types/markdown-it": "*",
        "markdown-it": "*"
```

This was needed for pnpm to work properly, i am going to leave it (no harm).